content/en/apps/tutorials/application-settings.md

```diff
@@ -0,0 +1,192 @@
+---
+title: "CHT Application Settings"
+linkTitle: Application Settings
+weight: 1
+description: >
+  Managing CHT application settings
+relatedContent: >
+  apps/reference/app-settings
+  apps/guides/performance/replication/#depth
+  core/overview/transitions 
+---
+
+## Purpose of the Tutorial
+
+This tutorial will take you through how to manage the CHT application settings, including;
+
+- Setting user roles and permissions
+- Enabling and disabling transitions
+- Configuring contact hierarchy and configuring replication.
+
+App settings allow you to both persist information that is critical to the application outside the code, and to create profiles that store the preferences for project deployments.
+
+## Brief Overview of Key Concepts
+
+The settings which control CHT apps are defined in the *app_settings.json* file, and stored in the settings doc in the database.
+
+*Permissions* are settings that control access to specific app features and functionality.
+
+*Roles* define permissions for users to perform access a group of app features and functionality.
+
+*Replication* is when users download a copy of the data on to their device. *[Replication depth](https://docs.communityhealthtoolkit.org/apps/guides/performance/replication/#depth)* refers to the number of levels within a hierarchy a specific user role is able to replicate.
+
+*[A transition](https://docs.communityhealthtoolkit.org/core/overview/transitions/)* is javascript code that runs when a document is changed. A transition can edit the changed doc or do anything server side code can do for that matter.
+
+## Required Resources
+
+You should have a functioning CHT instance and have medic-conf installed locally. {{< see-also page="apps/tutorials/local-setup" title="How to set up a CHT local configuration environment" >}}
+
+## Implementation Steps
+
+In this section, you will define a new role, set persmissions for the role, set transitions, configure a hierarchy, and upload your modified app settings file to your local environment.
+
+### 1. Set Roles and Permissions
+
+To add a new role, edit the object corresponding to `"roles"` key in `app_settings.json`. Add the new role as a key and within it, have an object with key/value pairs indicating the translation key of the role and whether it is an online or offline role.
+
+Configure a CHW role by adding the following snippet to the `"roles"` object:
+
+```json
+  "chw": {
+    "name": "usertype.chw",
+    "offline": true
+  }
+```
+
+Set permissions for the new role by adding the role to the relevant permission in the `"permissions"` object. For instance, to grant the CHW role permission to create people, add the role to the array with the key `"can_create_people"`.
+
```

Comment:
add a see also: {{< see-also page="apps/reference/app-settings/user-permissions" title="Permissions" >}}

note: this is a placeholder page currently, but should be updated soon

Reply:
Updated.